### PR TITLE
kind: add CNI flannel

### PR DESCRIPTION
**Pull request by dougsland**
Adding flannel files for running the tests via kind project.

Signed-off-by: Douglas Schilling Landgraf <dougsland@gmail.com>

### CODE REVIEW

File: jobs/flannel-job.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+metadata:
+  name: cyclonus
+spec:
+  template:
+    spec:
+      restartPolicy: Never
+      containers:
+        - command:
+            - ./cyclonus
+            - generate
```
Comment (mattfenwick) [inline jobs/flannel-job.yaml:12]: Based on our slack convo, you could increase the time cyclonus waits for pods to come up by adding an `--pod-creation-timeout-seconds=120` arg here.

**Review by mattfenwick — APPROVED** [review]
🎉 🎉 nice one @dougsland !!

**Review by mattfenwick — CHANGES_REQUESTED** [review]
Based on @rpkatz comment, Flannel doesn't support so netpols so there's no point in testing it!

**Comment by dougsland** [comment]
> Based on @rpkatz comment, Flannel doesn't support so netpols so there's no point in testing it!

Yeah, agreed. Let me prepare another patch.

**Pull request closed without merging**